k8s/helm/yona/templates/10_deployment_analysis.yaml

```diff
@@ -38,8 +39,8 @@ spec:
               value: "10s"
       containers:
         - name: analysis
-          image: 'yonadev/analysisservice:build-{{ .Chart.AppVersion }}'
-          imagePullPolicy: Always
+          image: '{{ .Values.global.imagebase | default "yonadev" }}/analysisservice:build-{{ .Chart.AppVersion }}'
+          #imagePullPolicy: Always
```

Comment:
Is this intentionally commented?

Reply:
Yes - some times I want it, so it is there to not have to look up the variable name.   It could be removed.

Reply:
OK